### ISSUE TITLE
Add Banned users to Banned list.

### ISSUE THREAD

**Issue by Firebottle** [Firebottle/Firebot#470]
With constellation and the chat stuff now in place, we have the ability to see when user's roles are updated. For example, when the "Banned" role is added to someone.

It'd be nice if we could automatically add these people to the banned userlist in Firebot. Right now, even if you ban someone they can still user interactive as far as I'm aware. This might also be an issue Mixer fixes before we can get to it.

**Comment by Firebottle** [comment]
Added this in to the dev branch.

If a user update comes in over constellation we check the roles and if "Banned" is in there then we check the ban list and add them if they're not already there.

We DONT however, unban them automatically right now. The reason being, constellation doesnt tell us what roles were changed, we just get a full list. So if I manually added someone to the banned userlist, and then they subscribed to the channel. We'd get a constellation update without the "Banned" event, and then it would try to pull them out of our list.

We can probably get around that, but for now I think we can assume that users banned from the channel should remain banned from interactive until manually fixed.
